.github/workflows/docker-image.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           images: |
             rob93c/stickerify
           tags: |
-            type=raw,value=latest,enable={{is_default_branch}}
+            type=ref,event=latest,enable=${{ github.ref != 'refs/heads/main' }}
```

Comment:
The actual change should be applied here, as described in the related issue.

Reply:
So basically you want do something like this --> type=raw,value=latest,enable=${{ github.ref != 'refs/heads/main' }}

Reply:
Yeah sorry, I commented on the wrong line, I meant the one below acrually

Reply:
Can you check now @MartelliEnrico.